docs-new/docs/getting-started/lifecycle-management.md

```diff
@@ -72,29 +72,7 @@ Add a task which will trigger after a deployment.
 Change `UUID` to whatever value you have.
 Apply this manifest:
 
-```yaml
----
-apiVersion: lifecycle.keptn.sh/v1alpha3
-kind: KeptnTaskDefinition
-metadata:
-  name: send-event
-  namespace: keptndemo
-spec:
-  retries: 0
-  timeout: 5s
-  container:
-    name: curlcontainer
-    image: curlimages/curl:latest
-    args: [
-        '-X',
-        'POST',
-        'http://webhook.webhook.svc.cluster.local:8084/YOUR-UUID-HERE',
-        '-H',
-        'Content-Type: application/json',
-        '-d',
-        '{ "from": "keptn send-event" }'
-    ] 
-```
+{% include "./assets/lifecycle-management.md_1.yaml" %}
```

Comment:
All the includes should be in the following format"

```suggestion
```yaml
{% include "./assets/lifecycle-management.md_1.yaml" %}
```
```